signac/contrib/project.py

```diff
@@ -372,7 +372,7 @@ def open_job(self, statepoint=None, id=None):
             # second best case
             job = self.Job(project=self, statepoint=statepoint)
             if job._id not in self._sp_cache:
-                self._sp_cache[job._id] = dict(job._statepoint)
+                self._sp_cache[job._id] = job._statepoint._as_dict()
```

Comment:
Is there a reason to use the `__call__` syntax in  `self._statepoint = SyncedAttrDict(dst._statepoint(), parent=_sp_save_hook(self))` and `_as_dict()` here? If I recall correctly, both do the same thing. I don't think I have a preference, just trying to understand for consistency.

Reply:
I needed to look this up. The `__call__()` function will also trigger a load-operation. So using `_as_dict()` would be correct here.